core/utils.js

```diff
@@ -36,31 +37,21 @@ const style = goog.require('Blockly.utils.style');
 const userAgent = goog.require('Blockly.utils.userAgent');
 
 
-/**
- * Don't do anything for this event, just halt propagation.
- * @param {!Event} e An event.
- */
-const noEvent = function(e) {
-  // This event has been handled.  No need to bubble up to the document.
-  e.preventDefault();
-  e.stopPropagation();
-};
-exports.noEvent = noEvent;
+Object.defineProperty(exports, 'noEvent', {
```

Comment:
Check with Chris about this, but I vaguely recall him mentioning that we should be using defineProperties instead (I think due to quoted field names not playing nicely with obfuscation? I also realized I might need to change this in a couple of my own recent PRs)

Reply:
@cpcallen

Reply:
Yes: `Object.defineProperties` is preferred to `Object.defineProperty` because the Closure Compiler in advanced optimization mode will successfully rename the properties supplied as keys in the argument object, whereas it will not touch a property name supplied as a string.

But to step back a level here: why are you providing  a get accessor here when it looks like a stub function would work just as well?:

```
/**
 * Don't do anything for this event, just halt propagation.
 * @deprecated Use Blockly.browserEvents.noEvent() instead.
 * @param {!Event} e An event.
 */
const noEvent = function(e) {
  deprecation.warn(
      'Blockly.utils.noEvent', 'September 2021', 'September 2022', /* etc */);
  browserEvents.noEvent(e);
};
exports.noEvent = noEvent;
```

(Or have I misunderstood what is going on here?)

Reply:
> why are you providing a get accessor here when it looks like a stub function would work just as well?

No reason, I just thought that's what I was supposed to do haha. I'll switch them over 👍